api/src/routes/user.ts

```diff
@@ -90,6 +106,119 @@ export const userRoutes: FastifyPluginCallbackTypebox = (
       }
     }
   );
+  fastify.get(
+    '/user/get-session-user',
+    {
+      schema: schemas.getSessionUser
+    },
+    async (req, res) => {
+      try {
+        // TODO: promise all to fetch both
+        const userToken = await fastify.prisma.userToken.findFirst({
+          where: { userId: req.session.user.id }
+        });
```

Comment:
I decided to return a `500` with `danger` flash here. I realise this is a divergence from the old API, but we are talking about a major bug.

Reply:
Maybe we want a flash, but still to present the older of the double-user.

Reply:
This is userTokens, though. They can have more than one in principle.

```
      const tokenInfo = await UserToken.findOne({
        where: { id: userToken }
      });

      if (!tokenInfo) return res.send('User token not found');

      const { userId } = tokenInfo;

      // check if user exists for user token
      const user = await User.findOne({
        where: { id: userId }
      });
```

is how they're used right now.